modules/ossm-tutorial-jaeger-generating-traces.adoc

```diff
@@ -33,8 +33,13 @@ This tutorial uses {ProductShortName} and the Bookinfo tutorial to demonstrate h
 .. In the {product-title} console, navigate to *Networking* -> *Routes* and search for the Jaeger route, which is the URL listed under *Location*.
 .. Use the CLI to query for details of the route:
 +
+[NOTE]
+====
+Replace `<control_plane_project>` with the name of your control plane project. In this example, the control plane project is `istio-system`.
+====
++
 ----
-$ export JAEGER_URL=$(oc get route -n istio-system jaeger-query -o jsonpath='{.spec.host}')
+$ export JAEGER_URL=$(oc get route -n <control_plane_project> jaeger -o jsonpath='{.spec.host}')
```

Comment:
@JStickler +1 on the above response. 
But `istio-system` is removed from the example command. That's my concern. Thanks!

Reply:
I added the note and the `<control_plane_project>` placeholder for consistency since it's used earlier in the doc [1].

Should both examples be updated with `istio-system` instead?

[1] - [modules/ossm-tutorial-bookinfo-install.adoc](https://github.com/openshift/openshift-docs/blob/master/modules/ossm-tutorial-bookinfo-install.adoc#L84)

Reply:
Yes, we should probably update all of the examples to be consistent.

Reply:
There's a ticket (OSSMDOC-315) to update the examples, so I'm going to go ahead and merge this PR as we have a customer waiting for this update.